build_tools/github_actions/configure_ci.py

```diff
@@ -8,43 +8,76 @@
 """Determines whether CI should run on a given PR.
 
 The following environment variables are required:
-- GITHUB_EVENT_NAME: GitHub event name, e.g. pull_request.
-- GITHUB_OUTPUT: path to write workflow output variables.
-- GITHUB_STEP_SUMMARY: path to write workflow summary output.
+"GITHUB_EVENT_NAME",: GitHub event name, e.g. pull_request.
+"GITHUB_OUTPUT",: path to write workflow output variables.
+"GITHUB_STEP_SUMMARY",: path to write workflow summary output.
```

Comment:
remove commas or colons?

`"VARIABLE",:` could just be `"VARIABLE":` or `"VARIABLE",`

```suggestion
"GITHUB_EVENT_NAME": GitHub event name, e.g. pull_request.
"GITHUB_OUTPUT": path to write workflow output variables.
"GITHUB_STEP_SUMMARY": path to write workflow summary output.
```

Reply:
Uh, I have no idea why this happened. I definitely didn't do this intentionally